.travis.yml

```diff
@@ -12,12 +12,8 @@ jobs:
     - php: 7.1
     - php: 7.2
     - php: 7.3
+    - php: 7.4
       env: WITH_COVERAGE=true
-    - php: nightly
-
-matrix:
-    allow_failures:
-        - nightly
```

Comment:
I think we should still allow failures on nightly

Reply:
Adjusted!